code/_onclick/adjacent.dm

```diff
@@ -88,9 +88,23 @@
 //Multitile special cases.
 /obj/structure/Adjacent(atom/neighbor)
 	if(bound_width > 32 || bound_height > 32)
```

Comment:
how comes we don't have a flag for multi loc objs?

Reply:
bound width and height need to be >32 for multitile and thats a byond builtin